File: MT.Model/Dictionary/GrantRatio.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace MT.Model
{
    #region GrantRatio:EntityBase
    /// <summary>
    /// This object represents the properties and methods of a GrantRatio:EntityBase.
    /// </summary>
    public class GrantRatio : EntityBase
    {
        [Key]
        public Guid GrantRatioID { get; set; }
        public string GrantRatioCode { get; set; }
        public string GrantRatioName { get; set; }
        public string Description { get; set; }
        public bool Inactive { get; set; }
        public int SortOrder { get; set; }
        public int Rownum { get; set; }
    }
    #endregion
}

File: BECore/app/model/GrantRatio.js
/// <summary>
/// Model EXTJS GrantRatio.
/// </summary>
Ext.define('QLDT.model.GrantRatio', {
    extend: 'Ext.data.Model',
    idProperty: 'GrantRatioID',
    idType: 'Guid',

    code: 'GrantRatioCode',
    name: 'GrantRatioName',

    fields: [
		{ name: "GrantRatioID", type: "string" },
		{ name: "GrantRatioCode", type: "string" },
		{ name: "GrantRatioName", type: "string" },
		{ name: "Description", type: "string" },
		{ name: "Inactive", type: "boolean" },
		{ name: "SortOrder", type: "int" },
		{ name: "CreatedDate", type: "date" },
		{ name: "ModifiedDate", type: "date" },
		{ name: "IPAddress", type: "string" },
		{ name: "ModifiedBy", type: "string" },
		{ name: "CreatedBy", type: "string" },
        { name: "Rownum", type: "int" },
    ]
});

